### PR TITLE
Add battery api stuff

### PR DESCRIPTION
**Pull request by melneubert**

### CODE REVIEW

File: App/Containers/Device.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      devicePresent: false
+    }
+    this.fetchDeviceData()
+  }
+
+  async fetchDeviceData () {
+    let userId = await AsyncStorage.getItem(StorageKeys.FITBIT_USER_ID)
+    let accessToken = await AsyncStorage.getItem(StorageKeys.FITBIT_ACCESS_TOKEN)
+
+    fetch(`https://api.fitbit.com/1/user/${userId}/devices.json`, {
```
Comment (ipavl) [inline App/Containers/Device.js]: You can just use `-` for the current user's userId.

**Review by ipavl — COMMENTED** [review]
We might want to show the time the device synced in addition to just the date. Maybe add a test for the page, too.

**Comment by melneubert** [comment]
@ipavl Was going to add the tests in another pull request once we do some more UI update things. Fixed your other suggestions though.

**Review by ipavl — APPROVED** [review]

**Comment by mmacke01** [comment]
👍

**Pull request merged**